### ISSUE TITLE
Only Broadlink Accesories show up in homebridge but not homekit

### ISSUE THREAD

**Issue by jeremydvoss** [kiwi-cam/homebridge-broadlink-rm#668]
**Describe the bug**
The broadlink accessories do not show up in homekit despite showing up in homebridge and the Homebridge hub showing up in hoomekit. Triggering commands from the homebridge UI works.

Config:
```
{
    "bridge": {
        "name": "Homebridge XXXX Wifi 2",
        "username": "0E:A2:EA:20:XX:XX",
        "port": 51125,
        "pin": "496-27-115",
        "advertiser": "bonjour-hap",
        "bind": [
            "Wi-Fi 2"
        ]
    },
    "accessories": [],
    "platforms": [
        {
            "name": "Config",
            "port": 8581,
            "platform": "config"
        },
        {
            "platform": "BroadlinkRM",
            "name": "BroadlinkRM",
            "hideScanFrequencyButton": false,
            "hideLearnButton": false,
            "hideWelcomeMessage": true,
            "hosts": [
                {
                    "address": "192.168.4.XXX",
                    "mac": "ec:0b:ae:3d:XX:XX",
                    "isRFSupported": "true",
                    "isRM4": "true"
                },
                {
                    "address": "192.168.4.XXX",
                    "mac": "ec:0b:ae:a0:XX:XX",
                    "isRFSupported": "true",
                    "isRM4": "true"
                }
            ],
            "accessories": [
                {
                    "name": "Air Purifier",
                    "type": "outlet",
                    "data": {
                        "on": "...",
                        "off": "..."
                    },
                    "resendHexAfterReload": true,
                    "allowResend": true
                },
				...
				{
                    "name": "LG Projector",
                    "type": "tv",
                    "subType": "stb",
                    "pingIPAddress": "192.168.4.XXX",
                    "pingFrequency": 5,
                    "pingIPAddressStateOnly": true,
                    "pingUseArp": false,
                    "pingGrace": 15,
                    "data": {
                        "on": "...",
                        "off": "...",
                        "remote": {
                            ...
						},
                        ...
                        "inputs": [
                            ...
                        ]
                    }
                },
                {
                    "name": "RM Pro 2 Learn",
                    "type": "learn-code",
                    "host": "192.168.4.XXX"
                },
                {
                    "name": "RM Pro 1 Learn",
                    "type": "learn-code",
                    "host": "192.168.4.XXX"
                }
            ],
            "_bridge": {
                "username": "0E:BD:A4:95:34:91",
                "port": 30328
            }
        },
        ...
    ],
    "disabledPlugins": [
        ...
    ]
}
```

Accessories:
![Accesories](https://github.com/kiwi-cam/homebridge-broadlink-rm/assets/4807316/4cf30b84-de1c-4e4e-9546-8b924c8c013c)

Homekit bridges (Note that the broadlink rm pros used to show up as bridges _in addition_ to the homebridge server itself. I removed these as part of a system restart. However, when I added the new homebridge server back, These broadlink bridges never showed up):
![image](https://github.com/kiwi-cam/homebridge-broadlink-rm/assets/4807316/bcbeb15e-3669-41a2-9706-6fb2b02fd19c)

Homebridge logs (I will upload the full doc later after I make sure there is no personal info):
```
...
Enter this code with your HomeKit app on your iOS device to pair with Homebridge:
                       
    ┌────────────┐     
    │ 496-27-115 │     
    └────────────┘     
                       
[11/29/2023, 7:11:39 PM] Homebridge v1.7.0 (HAP v0.11.1) (Homebridge CEDF Wifi 2) is running on port 51125.
[11/29/2023, 7:11:39 PM] [BroadlinkRM] Launched child bridge with PID 18816
[11/29/2023, 7:11:41 PM] Registering platform 'homebridge-broadlink-rm-pro.BroadlinkRM'
[11/29/2023, 7:11:41 PM] [BroadlinkRM] Loaded homebridge-broadlink-rm-pro v4.4.15 child bridge successfully
[11/29/2023, 7:11:41 PM] Loaded 0 cached accessories from cachedAccessories.0EBDA4953491.
[11/29/2023, 7:11:41 PM] [BroadlinkRM] [INFO] Automatic Broadlink RM device discovery has been disabled as the "hosts" option has been set.
[INFO] Adding RF Support to device ec:0b:ae:3d:XX:XX with type 2227
[INFO] Adding RF Support to device ec:0b:ae:a0:XX:XX with type 2227
[11/29/2023, 7:11:41 PM] [BroadlinkRM] [INFO] Running Homebridge Broadlink RM Plugin version 4.4.15
[11/29/2023, 7:11:41 PM] [BroadlinkRM] LG Projector Accessory Ready
[11/29/2023, 7:11:41 PM] [BroadlinkRM] 
[11/29/2023, 7:11:41 PM] [BroadlinkRM] **************************************************************************************************************
[11/29/2023, 7:11:41 PM] [BroadlinkRM] You added TVs in your configuration!
[11/29/2023, 7:11:41 PM] [BroadlinkRM] Due to a HomeKit limitation you need to add any TVs to the Home app by using the Add Accessory function.
[11/29/2023, 7:11:41 PM] [BroadlinkRM] There you'll find your TVs and you can use the same PIN as you using for this HomeBridge instance.
[11/29/2023, 7:11:41 PM] [BroadlinkRM] **************************************************************************************************************
[11/29/2023, 7:11:41 PM] [BroadlinkRM] 
[11/29/2023, 7:11:41 PM] Initializing platform accessory 'Air Purifier'...
[11/29/2023, 7:11:41 PM] Initializing platform accessory 'Lamp'...
[11/29/2023, 7:11:41 PM] Initializing platform accessory 'Media Console Grow Lamps'...
[11/29/2023, 7:11:41 PM] Initializing platform accessory 'Projector Screen'...
[11/29/2023, 7:11:41 PM] Initializing platform accessory 'RM Pro 2 Learn'...
[11/29/2023, 7:11:41 PM] Initializing platform accessory 'RM Pro 1 Learn'...
[11/29/2023, 7:11:41 PM] Homebridge v1.7.0 (HAP v0.11.1) (BroadlinkRM) is running on port 30328.
[11/29/2023, 7:11:41 PM] LG Projector 1452 is running on port 55234.
[11/29/2023, 7:11:41 PM] Please add [LG Projector 1452] manually in Home app. Setup Code: 496-27-115
[11/29/2023, 7:11:41 PM] [BroadlinkRM] [INFO] Discovered Manual RM4 Pro Device (2227) at 192.168.4.XXX (ec:0b:ae:a0:XX:XX)
[11/29/2023, 7:11:42 PM] [BroadlinkRM] [INFO] Discovered Manual RM4 Pro Device (2227) at 192.168.4.XXX (ec:0b:ae:3d:XX:XX)
...
[11/29/2023, 7:11:43 PM] [BroadlinkRM] Air Purifier setSwitchState: false
[11/29/2023, 7:11:43 PM] [BroadlinkRM] Air Purifier sendHex (192.168.4.223; ec:0b:ae:a0:XX:XX) 
...
[11/29/2023, 7:11:43 PM] [BroadlinkRM] Lamp setSwitchState: false
[11/29/2023, 7:11:43 PM] [BroadlinkRM] Lamp sendHex (192.168.4.223; ec:0b:ae:a0:XX:XX)
...
[11/29/2023, 7:11:43 PM] [BroadlinkRM] Media Console Grow Lamps setSwitchState: false
[11/29/2023, 7:11:43 PM] [BroadlinkRM] Media Console Grow Lamps sendHex (192.168.4.223; ec:0b:ae:a0:XX:XX)
...
[11/29/2023, 7:11:43 PM] [BroadlinkRM] Projector Screen setSwitchState: false
[11/29/2023, 7:11:43 PM] [BroadlinkRM] Projector Screen sendHex (192.168.4.223; ec:0b:ae:a0:XX:XX)
...
[11/29/2023, 7:11:44 PM] [BroadlinkRM] Air Purifier Accessory Ready
[11/29/2023, 7:11:44 PM] [BroadlinkRM] Lamp Accessory Ready
[11/29/2023, 7:11:44 PM] [BroadlinkRM] Media Console Grow Lamps Accessory Ready
[11/29/2023, 7:11:44 PM] [BroadlinkRM] Projector Screen Accessory Ready
...
```

**To Reproduce**
Unknown. This server used to work and randomly broke.

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS: WIndows 10
 - Browser Firefox

**Smartphone (please complete the following information):**
 - Device: iphone 15
 - OS: iOS
 - Browser safari

**Additional context**
Homebridge 1.7.0
homebridge-broadlink-rm-pro v4.4.15 
Node.js Version: v20.10.0

**Comment by jeremydvoss** [comment]
Possibly related issue, when I try to add the Projector manually, it says the device is already added since it has the same ID as homebridge itself.

**Comment by jeremydvoss** [comment]
Think I solved everything. I think the main bug either had to do with the child bridges or the "bind" option that appeared in the config when I switched wifi adapters. After removing those and going to a far simpler config, most devices showed up.

The tv still wouldn't, to solve that, I just renamed it to "LG Projector 2". So, the way we add tv maybe have some bug whete they can't be added again.
